CONTRIBUTING.md

```diff
@@ -368,6 +369,27 @@ at the beginning of the line:
 I love it! Very clean code and great tests. lgtm.
 ```
 
+## Continuous Integration
+
+The Kata Containers project has a gaiting process to prevent introducing
+regressions. When your PR is submitted, a Continuous Integration (CI) system
+will run different checks on different platforms for your changes. Currently
+we use [Jenkins](http://jenkins.katacontainers.io) and
+[travis](https://travis-ci.org/kata-containers/) for testing your changes.
+
+Some of the checks are:
+
+- Static analysis checks.
+- Unit tests.
+- Functional tests.
+- Integretation tests.
+
+The travis job will be executed right after the PR is opened, while the Jenkins
+jobs will wait to be triggered. A maintainer must add a `/test` comment
```

Comment:
btw @chavafg I think I have noticed that the `/test` has to be maybe the only line, or the first line, of a comment. I have no proof, but that is how it felt. That is, adding a '/test' at the end of a comment for instance I don't think is triggering a build. I'm not sure if that is true or if it is something we can control or fix in the Jenkins plugin. But, if it is true, we may need to document that here for clarity.

Reply:
checking this... I think it can be solved in the plugin.

Reply:
fixed

Reply:
@chavafg Is it possible to trigger a specific type of build? I mean maybe the yum repos were down the first time we ran the tests but all others passed. In such a case we don't want to rerun all the tests again. Is there someway we just run the fedora tests again?

Reply:
hi @ydjainopensource - oops, nobody answered I think :-) In theory it would be possible I think to set up regexp rebuild patterns on the jobs to allow triggering of individual or groups of builds. @chavafg something for us to consider as a later addition maybe (I think we have already considered adding one just for metrics for instance).